### PR TITLE
chance to get the standard red spacesuit from syndicate space suit boxes

### PR DESCRIPTION
**Pull request by ghost**
:cl: asdfsdagasdghbdsagsdffgd
add: chance to get either the soulless black and red space suit or the soul red spacesuit
/:cl:

[why]: the red syndie spacesuit is way underused and i feel like i want to feel special when buying the space suit from the uplink

first pr that _isn't_ removal

### CODE REVIEW

File: code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -206,8 +206,14 @@
 	STR.can_hold = typecacheof(list(/obj/item/clothing/suit/space/syndicate, /obj/item/clothing/head/helmet/space/syndicate))
 
 /obj/item/storage/box/syndie_kit/space/PopulateContents()
-	new /obj/item/clothing/suit/space/syndicate/black/red(src) // Black and red is so in right now
-	new /obj/item/clothing/head/helmet/space/syndicate/black/red(src)
+	switch (pickweight(list("goon" = 1, "normal" = 1)))
```
Comment (coiax) [inline code/game/objects/items/storage/uplink_kits.dm]: No need for this, just put it in an `if(prob(50))` statement.

**Comment by tuypo1** [comment]
Why not make it an extra option that costs one telecrystal more instead

**Comment by duckay** [comment]
> Why not make it an extra option that costs one telecrystal more instead

seemed unnecessary as it performs the same purpose

File: code/game/objects/items/storage/uplink_kits.dm (hunk 1 above)
Comment (duckay) [reply code/game/objects/items/storage/uplink_kits.dm]: if i'm gonna be honest im not sure where to put that

Comment (coiax) [reply code/game/objects/items/storage/uplink_kits.dm]: `if(prob(50)
    do_stuff()
else
    something_else()`

**Comment by ExcessiveUseOfCobblestone** [comment]
isn't this just a vanity change?

**Comment by duckay** [comment]
> isn't this just a vanity change?

is it?

**Comment by ExcessiveUseOfCobblestone** [comment]
I hate you for making me look it up but yes it is. Was curious since he said make it cost another TC lol.

You could just make it spawn a choice beacon and allow the player to select any of the subtypes

**Comment by duckay** [comment]
> I hate you for making me look it up but yes it is. Was curious since he said make it cost another TC lol.
> 
> You could just make it spawn a choice beacon and allow the player to select any of the subtypes

while i'd love that i can barely code

**Comment by ExcessiveUseOfCobblestone** [comment]
It could be modified to not use the pod.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**